src/movies.js

```diff
@@ -1,13 +1,34 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+function getAllDirectors(movies) {
+    const myArr = movies.filter(function (director) {
+      });
+      return myArr
+      console.log(myArr);
+      const myArr1 = movies.map(function (director) {
+        return myArr1
+
+      });
+      console.log(myArr1);
+}
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
-function howManyMovies(moviesArray) {}
+function howManyMovies(moviesArray) {
+    const arr = movies.filter(function(director,genre) {
+     
+        return director.name == "Steven Spielberg"
+        return genre.name == "Drama"
+        return arr
```

Comment:
Remember that a function usually can have just one `return` the exception would be if you have different triggers for thar return to be reached

```
	function biggerThanOne(number) {
		if (number > 1) {
			return 'The number is bigger than one'
		} else {
			return 'The number is smaller than one'
		}
	}
```

In the previous case you only have one return being activate anytime you call the function 
biggerThanOne(2) => return 'The number is bigger than one'
biggerThanOne(0) => return 'The number is smaller than one'